AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -188,6 +188,9 @@ www.kinopoisk.ru##footer > div:has(> div > div[data-testid="kp-ui.amediateka-foo
 !
 ! SECTION: Self-promo - Regular rules
 !
+brusselstimes.com#$#.tp-modal:has(> div.tp-iframe-wrapper > iframe[src^="https://api-eu.piano.io/"]) { display: none !important; }
+brusselstimes.com#$#.tp-backdrop.tp-active { display: none !important; }
+brusselstimes.com#$#body.tp-modal-open { overflow: auto !important; position: static !important; touch-action: unset !important; background: unset !important; }
```

Comment:
All rules are good, but we have a group for TinyPass/Piano
https://github.com/AdguardTeam/AdguardFilters/blob/74f6714167a3d7df9d4d7a09b8b7b8571e3fc936/AnnoyancesFilter/Other/sections/self-promo.txt#L338-L340
better add domains there